chia/util/initial-config.yaml

```diff
@@ -254,9 +254,9 @@ full_node:
   # How often to initiate outbound connections to other full nodes.
   peer_connect_interval: 30
   # Accept peers until this number of connections
-  target_peer_count: 60
+  target_peer_count: 80
   # Initiate outbound connections until this number is hit.
-  target_outbound_peer_count: 10
+  target_outbound_peer_count: 8
```

Comment:
I prefer to keep it at 10, due to:
1. Security assumptions. Assuming at least 1/10 peers is not an attacker is a smaller assumption than 1/8.
2. Fan-out: more fanout means faster propagation of blocks, which can become an issue if block validation speed is not fast.
3. What if you get unlucky and all your peers are slow or not synced?

Reply:
I agree, 8*8*8=512 while 10*10*10=1000 thats a big difference in fan out for a very common case (outgoing only)

Reply:
Also weak machines will not be able to deal with 80 connections. I've lowered mine to 20.

Reply:
Fan out should be calculated as follows:
(our_connections) * (max_outbound + 0.125 * max_inbound) ^ steps

Reply:
https://docs.google.com/spreadsheets/d/1MlWML6LPzmRGEJgpWgLCxWmdnbmlsP0YYgcuD8EaPZk/edit#gid=0

Reply:
0.125 is fraction of nodes that have open port or upnp that works